openmdao/drivers/tests/test_genetic_algorithm_driver.py

```diff
@@ -353,6 +353,26 @@ def test_SimpleGADriver_missing_objective(self):
 
         self.assertEqual(exception.args[0], msg)
 
+    def test_vectorized_constraints(self):
+        prob = om.Problem()
+        model = prob.model
+
+        dim = 2
+        model.add_subsystem('x', om.IndepVarComp('x', np.ones(dim)), promotes=['*'])
+        model.add_subsystem('f_x', om.ExecComp('f_x = sum(x * x)', x=np.ones(dim), f_x=1.0), promotes=['*'])
+        model.add_subsystem('g_x', om.ExecComp('g_x = 1 - x', x=np.ones(dim), g_x=np.zeros(dim)), promotes=['*'])
+
+        prob.driver = om.SimpleGADriver()
+
+        prob.model.add_design_var('x', lower=-10, upper=10)
+        prob.model.add_objective('f_x')
+
+        # Without bugfix to genetic_algorithm_driver, using vectorized constraint limits causes an exception
+        prob.model.add_constraint('g_x', upper=np.zeros(dim))
+
+        prob.setup()
+        prob.run_driver()
+
```

Comment:
Maybe test that the constraint is satisfies (and not just the lack of exception)?
```
        for i in range(dim):
            self.assertLessEqual(prob["g_x"][i], 0.0)
```

Reply:
Addressed by d45b1fb3bdfd441c28e73da613f8fd240f6b44d3

Reply:
Note that I decided that check that x >= 1 instead of checking that g_x <= 0. However, I just realized I have flipped the inequality...

Reply:
...and this if fixed by b3201d37cabfe733f8d6b3e0a916a30451458d3a